src/Orleans/Logging/LoggerImpl.cs

```diff
@@ -240,13 +240,12 @@ private void WriteLogMessage(int errorCode, Severity sev, string format, object[
             {
                 try
                 {
-                    consumer.Log(sev, loggerType, Name, message, LogManager.MyIPEndPoint, exception, errorCode);
+                    consumer.Log(sev, Name, message, exception, errorCode);
```

Comment:
Yes that's correct. I Haven't made that change in this PR. Orleans.Logger will become a wrapper on top of ASP.Net ILogger . Is this what you are asking?